aruco_ros/src/marker_publish.cpp

```diff
@@ -44,9 +45,18 @@ or implied, of Rafael Muñoz Salinas.
 #include <sensor_msgs/image_encodings.h>
 #include <aruco_ros/aruco_ros_utils.h>
 #include <aruco_msgs/MarkerArray.h>
+#include <geometry_msgs/Point.h>
 #include <tf/transform_listener.h>
+#include <tf/transform_broadcaster.h>
 #include <std_msgs/UInt32MultiArray.h>
 
+string int_to_string( const int a )
```

Comment:
Please handle this in the minimal invasive way.
This seems like a good alternative to adding a new function: http://www.boost.org/doc/libs/1_55_0/doc/html/boost_lexical_cast/examples.html